### PR TITLE
feat(ai): add encoder to language model info

### PR DESCRIPTION
**Pull request by klcd**

### CODE REVIEW

File: unique_toolkit/unique_toolkit/language_model/infos.py [hunk 1]
```diff
@@ -20,6 +20,35 @@ class LanguageModelName(StrEnum):
...
+    O200K_BASE = "o200k_base"
+    CL100K_BASE = "cl100k_base"
+
+
+def get_encoder_name(model_name: LanguageModelName) -> Optional[EncoderName]:
+    LMN = LanguageModelName
+    match model_name:
+        case (
+            LMN.AZURE_GPT_35_TURBO
+            | LMN.AZURE_GPT_35_TURBO_16K
```
Comment (abiodun0) [inline unique_toolkit/unique_toolkit/language_model/infos.py:33]: Would or be better instead of `|`?

**Review by abiodun0 — APPROVED** [review]

File: unique_toolkit/unique_toolkit/language_model/infos.py (hunk 1 above)
Comment (klcd) [reply unique_toolkit/unique_toolkit/language_model/infos.py:33]: `or` gives me a syntax error 😮

Comment (klcd) [reply unique_toolkit/unique_toolkit/language_model/infos.py:33]: Maybe it is not implement for the match flow

Comment (abiodun0) [reply unique_toolkit/unique_toolkit/language_model/infos.py:33]: Ah, my bad then. I think this looks good.

**Pull request merged**